packages/notification-center/src/hooks/use-notifications.hook.ts

```diff
@@ -31,7 +31,7 @@ export function useNotifications(props?: IUseNotificationsProps) {
     await mapFetchNextPage(storeId);
   }
 
-  const hasNextPage = mapHasNextPage?.has(storeId) ? mapHasNextPage.get(storeId) : true;
+  const hasNextPage = mapHasNextPage[storeId] ? mapHasNextPage[storeId] : false;
```

Comment:
@scopsy @ainouzgali 
Any idea for a reason why to return here a `true` in the end?

Reply:
@djabarovgeorge I think that initialy when no information loaded we need to do the first query to get the data and hence it was true? But I'm not 100% sure it's actually needed.

Reply:
No, I even think it could be removed. As it is supposed to create a default store id if not using store. Same as notifications